elf_ctx.c

```diff
@@ -166,12 +199,51 @@ void elf_ctx_init(struct ukarch_ctx *ctx, struct elf_prog *prog,
 		{ AT_PHNUM, prog->phdr.num },
 		{ AT_PHDR, (__uptr)prog->vabase + prog->phdr.off },
 #if CONFIG_APPELFLOADER_VDSO
+		/* TODO: This must also be pushed and copied
```

Comment:
The current state of the VDSO is unclear and since its size is not that big I would rather remap it instead of memcpy-ing it in a future PR, so I added it as a REVISIT on  my TODO list instead.

Reply:
In a single AS, can't we keep it to stay just one VDSO image? We can mark it as `ro`.